admin_ui/src/components/BulkUpdateModal.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+    <Modal v-on:close="$emit('close')">
+        <template v-if="schema">
+            <p>Select a column to update</p>
+
+            <select name="property" v-model="propertyName">
+                <option :value="schema.primary_key_name">
+                    Please select column
+                </option>
+                <option
+                    :key="key"
+                    :value="key"
+                    v-for="(property, key) in schema.properties"
+                >
+                    {{ key }}
+                </option>
+            </select>
+            <p v-show="propertyName != schema.primary_key_name">
+                Value to change
+            </p>
+
+            <form v-on:submit.prevent="updateRows($event)">
+                <div :key="key" v-for="(property, key) in schema.properties">
+                    <InputField
+                        v-if="key == propertyName"
+                        v-bind:format="property.format"
+                        v-bind:isFilter="isFilter"
+                        v-bind:key="property.title"
+                        v-bind:title="property.title"
+                        v-bind:type="property.type || property.anyOf[0].type"
+                        v-bind:choices="property.extra.choices"
+                    />
+                </div>
+                <button>Update</button>
+            </form>
+        </template>
+    </Modal>
+</template>
+
+
+<script lang="ts">
+import InputField from "../components/InputField.vue"
+import Modal from "../components/Modal.vue"
+import * as i from "../interfaces"
+
+export default {
+    props: {
+        schema: Object,
+        tableName: String,
+        selectedRows: {
+            type: Array,
+            defult: () => []
+        },
+        isFilter: {
+            type: Boolean,
+            default: false
+        }
+    },
+    components: {
+        Modal,
+        InputField
+    },
+    data() {
+        return {
+            propertyName: this.schema.primary_key_name
+        }
```

Comment:
I"m not sure about this - setting it to the primary key name is a bit confusing. Can it just default to null instead?

Reply:
We can't change that because we need that value to be able to hide the `Value to change`.
```Vue
<p v-show="propertyName != schema.primary_key_name">
    Value to change
</p>
```

Reply:
Will this work:

```vue
<p v-show="propertyName">
    Value to change
</p>
```

Reply:
Yes. That works if I set `propertyName: null` but I need to remove `Please select column` option.